Backpack-SwiftUI/Rating/Classes/BPKRating.swift

```diff
@@ -0,0 +1,129 @@
+            HStack(alignment: .lastTextBaseline, spacing: .zero) {
+                BPKText(
+                    ratingScale.displayedValue(from: value),
+                    style: size.fontStyle.ratingValueLabelFontStyle
+                ).foregroundColor(.textPrimaryColor)
+                if showScale {
+                    BPKText(
+                        ratingScale.displayedScale(),
+                        style: size.fontStyle.ratingScaleLabelFontStyle
+                    ).foregroundColor(.textSecondaryColor)
+                }
+            }
+            if size == .large {
+                VStack(alignment: .leading, spacing: .zero) {
+                    titleViewBuilder()
+                    subtitleView
+                }
+            } else if size == .default {
+                HStack(alignment: .center, spacing: BPKSpacing.md.value) {
+                    titleViewBuilder()
+                    subtitleView
+                }
+            }
+        }
+        .accessibilityElement()
+        .accessibilityLabel(accessibilityLabel)
+    }
+
+    public init(
+        accessibilityLabel: String,
+        value: Float,
+        ratingScale: Scale = .zeroToFive,
+        size: Size = .default,
+        subtitle: String? = nil,
+        showScale: Bool = true,
+        @ViewBuilder titleViewBuilder: @escaping () -> AnyView
+    ) {
+        self.accessibilityLabel = accessibilityLabel
+        self.value = value
+        self.ratingScale = ratingScale
+        self.size = size
+        self.subtitle = subtitle
+        self.showScale = showScale
+        self.titleViewBuilder = titleViewBuilder
+    }
+
+    public init(
+        accessibilityLabel: String,
+        title: String,
+        value: Float,
+        ratingScale: Scale = .zeroToFive,
+        size: Size = .default,
+        subtitle: String? = nil,
+        showScale: Bool = true
+    ) {
+        self.accessibilityLabel = accessibilityLabel
+        self.value = value
+        self.ratingScale = ratingScale
+        self.size = size
+        self.subtitle = subtitle
+        self.showScale = showScale
+
+        self.titleViewBuilder = {
+            AnyView(
+                BPKText(title, style: size.fontStyle.titleLabelFontStyle)
+                    .foregroundColor(.textPrimaryColor)
+            )
+        }
+    }
+
+    @ViewBuilder
+    private var subtitleView: some View {
+        if let subtitle = subtitle {
+            BPKText(
+                subtitle,
+                style: size.fontStyle.subtitleLabelFontStyle
+            )
+            .foregroundColor(.textSecondaryColor)
+            .lineLimit(1)
+            .fixedSize()
+        } else {
+            EmptyView()
+        }
+    }
+}
+
+struct BPKRating_Previews: PreviewProvider {
+    static var previews: some View {
+        BPKRating(
+            accessibilityLabel: "Rated Excellent, 4.5 out of 5, Based on 1,532 reviews",
```

Comment:
I wonder if we really want the label as initializer.
in SwiftuI we can just add

.accessibilityLabel( ) as a modifier

Reply:
accessibilityLabel is removed in the latest commit 66938f88b6bb313d15dc94fbc2c6d7f6a128afc2